### PR TITLE
Update groups to passwordfile in authentication.md

### PR DESCRIPTION
**Pull request by xilabao**
ref to https://github.com/kubernetes/kubernetes/pull/39651

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2212)

### CODE REVIEW

File: docs/admin/authentication.md [hunk 1]
```diff
@@ -116,10 +116,11 @@ authentication is currently supported for convenience while we finish making the
 more secure modes described above easier to use.
 
 The basic auth file format is implemented in `plugin/pkg/auth/authenticator/password/passwordfile/...`
-and is a csv file with 3 columns: password, user name, user id.
+and is a csv file with 3 columns: password, user name, user id, followed by
```
Comment (deads2k) [inline docs/admin/authentication.md]: with 4 columns.

**Comment by deads2k** [comment]
@devin-donnelly this is for 1.6.  Is master open for that yet?

**Comment by xilabao** [comment]
@deads2k PTAL

**Comment by deads2k** [comment]
lgtm

**Pull request merged**